src/plugins/console/public/lib/row_parser.ts

```diff
@@ -28,8 +28,7 @@ export const MODE = {
   BETWEEN_REQUESTS: 32,
 };
 
-// eslint-disable-next-line import/no-default-export
-export default class RowParser {
+export class RowParser {
```

Comment:
`default` exports aren't removed?

Reply:
We disallow it and I cleaned it up as part of the removal in mappings - but restored as that can happen separate from this PR.